extension/src/uddsketch.rs

```diff
@@ -582,6 +581,46 @@ pub fn uddsketch_approx_percentile<'a>(percentile: f64, sketch: UddSketch<'a>) -
     )
 }
 
+#[pg_operator(immutable, parallel_safe)]
+#[opname(->)]
+pub fn arrow_uddsketch_approx_percentile_array<'a>(
+    sketch: UddSketch<'a>,
+    percentiles: toolkit_experimental::AccessorPercentileArray<'a>,
+) -> Vec<f64> {
+    approx_percentile_slice(percentiles.percentile.as_slice(), sketch)
+}
+
+// Approximate the value at the given approx_percentile (0.0-1.0) for each entry in an array
+#[pg_extern(
+    immutable,
+    schema = "toolkit_experimental",
+    parallel_safe,
```

Comment:
Should we launch any new things claiming to be parallel-safe until we have a way to prove it?

Reply:
Good point, I will remove that for now

Reply:
Update: When I try to remove `parallel_safe` from the new accessor and approx_percentile_array functions, 212 of our 214 tests end up failing to run. I initially included `parallel_safe` since the new functionality is copied from the existing approx_percentile functions which are currently marked parallel_safe. Is that a good enough reason to keep these new ones as `parallel_safe` for now?